tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -273,14 +272,14 @@ class CompletionArgSuite extends BaseCompletionSuite {
       |  test($f@@)
       |}
       |""".stripMargin,
-    """|$$foo = 
-       |$$foo = ${1:???}, | = ${2:???}
-       |""".stripMargin,
+    s"""|$$$$foo =${' '}
+        |$$$$foo = $${1:???}, | = $${2:???}
+        |""".stripMargin,
     topLines = Option(2),
     compat = Map(
-      "3" -> """|$$foo = 
-                |$$foo = ${1:???}, $$bar = ${2:???}
-                |""".stripMargin
+      "3" -> s"""|$$$$foo =${' '}
```

Comment:
Is this needed? Why not leave `_` <- seems like something Intellij is forcing you to do :D